deebot_client/commands/json/pads_cleaning_interval.py

```diff
@@ -11,29 +12,28 @@
 class GetPadsCleaningInterval(CommandWithMessageHandling, MessageBodyDataDict):
     """Get pads cleaning interval command."""
 
-    name = "getPadsCleaningInterval"
+    name = "getWashInterval"
 
     @classmethod
     def _handle_body_data_dict(
         cls, event_bus: EventBus, data: dict[str, Any]
     ) -> HandlingResult:
         """Handle message->body->data and notify the correct event subscribers.
+
         :return: A message response
         """
-        event_bus.notify(
-            PadsCleaningIntervalEvent(PadsCleaningInterval(int(data["interval"])))
-        )
+        event_bus.notify(PadsCleaningIntervalEvent(int(data["interval"])))
         return HandlingResult.success()
 
 
 class SetPadsCleaningInterval(SetCommand):
```

Comment:
I chose PadsCleaningInterval for readibility, if it changes to Wash it will match the name Ecovacs uses but then it will be unclear wat it does don't you think?

Why is PadsCleaningInterval() not needed in PadsCleaningIntervalEvent()?

Reply:
The commands should have the same name as the ecovacs commands without any typos (they have made some). In the HA integration we can rename them if required but the commands in the library should follow ecovacs

Please change also the name of the event to match the new name